Makefile

```diff
@@ -158,7 +158,7 @@ api-protos := \
 
 plugin-protos := \
 	proto/spire/common/plugin/plugin.proto \
-	proto/spire/plugin/agent/svidstore/v1/svidstore.proto \
+	proto/spire/plugin/agent/svidstore/v1unofficial/svidstore.proto \
```

Comment:
Renaming this directory was required to prevent gRPC namespace collisions. Generated code for Go registers packages globally using the path to the proto, relative to the import path it was found under, which was the same for both the SPIRE and SDK versions. It won't impact the runtime compatibility with existing external plugins.